code/modules/reagents/chemistry/reagents/drinks/alcohol_reagents.dm

```diff
@@ -42,6 +42,8 @@
 
 /datum/reagent/consumable/ethanol/New(list/data)
 	if(LAZYLEN(data))
+		if(!isnull(data["timecreated"]))
+			data["timecreated"] = world.time
```

Comment:
Rather than instantiate this in `/New` which has the potential to hit reagents we want to keep "effective", I recommend doing this in `reaction_finished`:
```dm
/datum/chemical_reaction/drink/reaction_finished(datum/reagents/holder, datum/equilibrium/reaction, react_vol)
	. = ..()
	for(var/result_type in results)
		var/datum/reagent/result = locate(result_type) in holder.reagents_list
		if(!result) // ??
			return
	LAZYSET(result.data, timecreated, world.time)
```
This ensures we really only care about reagents mixed mid round